src/lib.rs

```diff
@@ -31,42 +62,14 @@ where
 }
 
 #[derive(Debug, Clone)]
-pub struct Artifacts {
```

Comment:
Upstream just unconditionally called `print_metadata` regardless - figured I'd just do it directly. Can clean it up if it's still required, but it felt a but superfluous.

Reply:
If i remember correctly, i think the rationale behind print_metadata was so that the user can output it himself if he has a weird use case. It kinda feels like a non existent edge case, so I'm ok with removing this.